frontend/package.json

```diff
@@ -60,7 +60,7 @@
     "leaflet-rastercoords": "^1.0.5",
     "leaflet.locatecontrol": "^0.81.1",
     "leaflet.offline": "^3.1.0",
-    "next": "^14.2.15",
+    "next": "14.2.21",
```

Comment:
Any reason why we restrict the version to this one in particular?

Reply:
The app is not ready yet for next15